packages/code-du-travail-frontend/pages/themes/[slug].js

```diff
@@ -37,65 +33,58 @@ class Theme extends React.Component {
   }
 
   render() {
-    const { theme = { children: [] }, query, pageUrl, ogImage } = this.props;
-
-    const isRootTheme = theme && !theme.slug;
+    const { theme = {}, query, pageUrl, ogImage } = this.props;
 
     if (!theme) {
-      return <NotFound />;
+      return (
+        <Layout>
+          <Metas
+            url={pageUrl}
+            title="Thème introuvable - Code du travail numérique"
+            description={`Explorez les contenus autour du thème ${theme.title}`}
+            image={ogImage}
+          />
+          <Section variant="white">
+            <Container>
+              <AlertWrapper>
+                <Alert variant="warning">Ce thème n’a pas été trouvé</Alert>
+              </AlertWrapper>
+            </Container>
+          </Section>
+        </Layout>
+      );
     }
 
     return (
-      <PageLayout>
+      <Layout>
         <Metas
           url={pageUrl}
-          title={`${theme.title || "Thèmes"} - Code du travail numérique`}
+          title={`${theme.title} - Code du travail numérique`}
           description={`Explorez les contenus autour du thème ${theme.title}`}
           image={ogImage}
         />
-        <Search />
         <ThemeBreadcrumbs theme={theme} />
         {theme.children && theme.children.length > 0 && (
           <Section variant="white">
             <Themes
-              isRoot={isRootTheme}
+              isRoot={false}
```

Comment:
I think isRoot=false should be the default here

Reply:
Yes